src/main/java/hypersquare/hypersquare/play/execution/CodeExecutor.java

```diff
@@ -41,7 +41,7 @@ public CodeExecutor(int plotId) {
     }
 
     public void halt() {
-        for (var r : running.values()) r.cancel();
+        for (var r : running.values());
```

Comment:
Is it intentional that halting code does not halt anymore?

Reply:
yes, it messes with cancelling events

Reply:
halt needs to halt